tests/FunctionalClientTest.php

```diff
@@ -2,7 +2,6 @@
 
 namespace Clue\Tests\React\Docker;
 
-use Clue\React\Block;
 use Clue\React\Docker\Client;
 use React\EventLoop\Loop;
```

Comment:
It looks like this import may no longer be needed (here and elsewhere), can you take a look at this? :+1: 

```suggestion
```

Reply:
I think it's still used here:

https://github.com/dinooo13/reactphp-docker/blob/8760d38e2fc419f8da18858e15917131ff99bafe/tests/FunctionalClientTest.php#L404

Reply:
Indeed, thanks for checking!